src/test/beatmaptest.cpp

```diff
@@ -62,6 +62,12 @@ TEST_F(BeatMapTest, Scale) {
 
     pMap->scale(Beats::THREEFOURTHS);
     EXPECT_DOUBLE_EQ(bpm / 2, pMap->getBpm());
+
+    pMap->scale(Beats::FOURTHIRDS);
```

Comment:
Oh you could also re-order three-halves to come after two-thirds so that each pair cancels out.